components/post_profile_picture/post_profile_picture.jsx

```diff
@@ -51,8 +51,13 @@ export default class PostProfilePicture extends React.PureComponent {
 
     getPostIconURL = (defaultURL, fromAutoResponder, fromWebhook) => {
         const {enablePostIconOverride, hasImageProxy, post} = this.props;
-        const postIconOverrideURL = post.props.override_icon_url;
-        const useUserIcon = post.props.use_user_icon;
+        const postProps = post.props;
```

Comment:
You aren't using this anywhere, but otherwise LGTM

Reply:
I meant to use that. Done fixed it